src/main/java/io/strimzi/kafka/metrics/KafkaMetricsCollector.java

```diff
@@ -122,10 +124,16 @@ private String metricName(MetricName metricName) {
                 .toLowerCase(Locale.ROOT);
     }
 
-    private static Labels labelsFromTags(Map<String, String> tags) {
+    protected static Labels labelsFromTags(Map<String, String> tags) {
```

Comment:
We don't need `protected`, we don't expect subclasses

Reply:
@mimaison I'm calling labelsFromTags in the test:
 @Test
    public void testLabelsFromTags() {
        Map<String, String> tags = new LinkedHashMap<>();
        tags.put("k-1", "v1");
        tags.put("k_1", "v2");

        Labels labels = KafkaMetricsCollector.labelsFromTags(tags);
        assertEquals("k_1", PrometheusNaming.sanitizeLabelName("k-1"));
        assertEquals("v1", labels.get("k_1"));
        assertEquals(1, labels.size());
    }
Is there another way around this? Thanks

Reply:
Is there another way around this? Thanks

Reply:
> I'm calling labelsFromTags in the test
How is this an issue?

Have you tried removing `protected` and see what happens? 
You may want to review https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html

Reply:
Yes it goes to package-private. I misunderstood your comment. I thought we wanted to keep it as `private`.